### ISSUE TITLE
some decissions for bundler 

### ISSUE THREAD

**Issue by Souvikns** [asyncapi/bundler#8]
I wanted to have two things discussed for the bundler 

1. Should we move to typescript?
2. Should we remove the CLI from this project and make it a standalone library?

> @derberg

**Comment by derberg** [comment]
I personally prefer JS over TS, but this is just a personal preference, I'm just old with my habits 😆 so up to you what you prefer. I'm more opinionated if it comes to my preference to push for composition and functions everywhere instead of classes and inheritance.

As if it comes to CLI, yeap, we should remove it and follow the same principle we have everywhere, so standalone lib and integration in existing tools like AsyncAPI CLI and Studio

**Comment by Souvikns** [comment]
😄 I was thinking of converting it into typescript, but I think it is better to stick with javascript once I have already started with it, and yeah will be removing the CLI in the next PR.

**Comment by derberg** [comment]
good call, it is not going to be a huge project, rewriting can be done later once we have ready solution, well tested, and rewriting can be then even a good project for initiatives like Google Summer of Code

**Comment by Souvikns** [comment]
closing this 💯
